frontends/web/src/components/forms/input.tsx

```diff
@@ -14,27 +14,38 @@
  * limitations under the License.
  */
 
-import { h } from 'preact';
-import { input, errorText, isTransparent } from './input.css';
+import { h, RenderableProps } from 'preact';
+import * as styles from './input.css';
+
+export interface Props {
+    children?: JSX.Element;
+    className?: string;
+    error: JSON;
```

Comment:
I guess `error` is optional as well? This would solve quite a few of your CI issues…